lib/conf_loader.rb

```diff
@@ -21,7 +21,16 @@ class ConfLoader
   # @api public
   def self.load(path, env)
     template = ERB.new File.new(path).read
-    environments = YAML.load template.result(binding)
+    source = template.result(binding)
+
+    environments =
+      begin
+        # Ruby >= 3.1
+        YAML.load(source, aliases: true)
+      rescue ArgumentError
```

Comment:
Maybe doing this more explicitly would be safer:

```
if Gem::Version.new(RUBY_VERSION).release >= Gem::Version.new('3.1.0')
  # ...
else
  # ...
end
```

Reply:
I went even further https://github.com/salemove/conf_loader/pull/9/commits/d221cc7bb956289b95cd682df8ec386b00405e81